spec/unit/dcv/configurators/carnegie_blacklight_configurator_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe Dcv::Configurators::CarnegieBlacklightConfigurator do
+  let(:config) do
+    Blacklight::Configuration.new
+  end
+  let(:configurator) { Class.new { include Dcv::Configurators::BaseBlacklightConfigurator }.new }
+
+  describe 'fulltext search configuration' do
+    context 'flag is default' do
+      before do
+        described_class.configure(config)
+      end
+
+      it 'only includes the field values in facet.field' do
+        expect(config.search_fields['fulltext_tesim']).to be_a Blacklight::Configuration::SearchField
```

Comment:
A background info question: if the flag was not working as expected, what value would config.search_fields['fulltext_tesim'] return?

Reply:
hmm if the default was not working it would presumably be nil. It's an added feature here that only applies in two contexts: the Carnegie site and the Carnegie Centennial site; the latter shares all of the configuration *except* the fulltext field. Semantically it seemed preferable to have the explicit behavior be to suppress the fulltext option.